Ada_Zangemann-cs.txt

```diff
@@ -186,784 +186,712 @@ Society should support those who respect others' freedom.
 
 [[I:ada-p03
 IMAGE: Matthias Kirschner, Sandra Brandstätter - ADA & ZANGEMANN -
-Příběh softwaru, skateboardů a malinové zmrzliny.
-Title in the middle, authors in the upper left.
-In the lower right, Ada's family's dog with gray fur and some white 
-patches and with a red collar with shiny yellow LEDs, which wags its 
-tail. 
+Příběh o softwaru, skateboardech a malinové zmrzlině.
+Titulek uprostřed, autoři nahoře vlevo.
+Ve spodu vpravo je Adin pes s šedivou srstí, několika bílými skvrnami
+a červeným obojkem se svítícími žlutými LED, který vrtí ocasem.
 ]]
 Matthias Kirschner
-Sandra Brandstätter
+Sandra Brandstätterová
 
 ADA & ZANGEMANN
 
-Příběh softwaru, skateboardů a malinové zmrzliny
+Příběh o softwaru, skateboardech a malinové zmrzlině.
 
 ## 4 ##
 ==== Slide 2
-[B]yla jednou jedna malá holčička, která se jmenovala Ada. Její
-rodina byla tak chudá, že se všechny jejich úspory vešly do sklenice na
-sušenky. Neměli dost peněz, aby mohli žít v pořádném domě. Proto, Ada žila se
-svojí maminkou a se svým mladším bráchou, Alanem, v chatě na kraji města
-poblíž vrakoviště.
+[B]yla jednou jedna malá holčička jménem Ada. Její rodina byla tak chudá, že všechny jejich úspory se vešly do plechovky od sušenek. Neměli peníze ani na to, aby mohli bydlet v opravdovém bytě. Místo toho bydlely Ada, její maminka a její malý bratr Alan v chatrči poblíž vrakoviště na kraji města.
```

Comment:
```suggestion
[B]yla jednou jedna malá holčička jménem Ada. Její rodina byla tak chudá, že všechny jejich úspory se vešly do plechovky od sušenek. Neměli peníze ani na to, aby mohli bydlet v opravdovém bytě. Místo toho bydlela Ada se svou maminkou a malým bratrem Alanem v chatrči poblíž vrakoviště na kraji města.
```
Přeformulovala bych tu poslední větu, protože mi není zřejmé, čí bratr Alan je, jestli Ady nebo maminky, když se tam opakuje to "její". Plus by mělo být "bydleli", protože v několikanásobném podmětu je i Alan, ale v navrhované verzi už je to jedno.

Reply:
Klidně to tak může být, jenom bych rád podotknul, že -y  je zde správně, viz https://prirucka.ujc.cas.cz/?id=601, část 2.1, odstavec 2.

Reply:
Jo, máš pravdu, zapomněla jsem, že je možné obojí, pokud je přísudek před podmětem.